ml-proto/test/block.wast

```diff
@@ -200,7 +200,7 @@
 
 ;; TODO(stack): move this elsewhere
 (module (func $type-break-num-vs-void
-  (block (i32.const 66) (br 0))
+  (block (drop (i32.const 66)) (br 0))
```

Comment:
It's not clear why this code needs changing, it would have worked with the AST without change. Could I ask why this is problematic for binaryen at present?